pyproject.toml

```diff
@@ -57,9 +57,15 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
+    "black",
+    "ensembl-py @ git+https://github.com/Ensembl/ensembl-py.git",
```

Comment:
Duplicated (I would remove the one below and leave this one to preserve the alphabetical order).